controllers/drpolicy_controller.go

```diff
@@ -131,6 +131,10 @@ func validateDRPolicy(ctx context.Context,
 	// If new DRPolicy and clusters are deleted, then fail reconciliation?
 	found := 0
 
+	if len(drpolicy.Spec.DRClusters) == 0 {
+		return ReasonValidationFailed, fmt.Errorf("value for DRCluster cannot be empty")
```

Comment:
(nit) message can be changed. Currently refers to a field name that is not `DRCluster`

Reword to: "missing DRClusters list in policy"

Reply:
reworded